UIforETW/UIforETWDlg.cpp

```diff
@@ -769,7 +774,7 @@ void CUIforETWDlg::OnBnClickedStarttracing()
 	else
 		UIETWASSERT(0);
 
-	std::wstring kernelProviders = L" Latency+POWER+DISPATCHER+DISK_IO_INIT+FILE_IO+FILE_IO_INIT+VIRT_ALLOC+MEMINFO";
+	std::wstring kernelProviders = L" Base+Latency+POWER+DISPATCHER+DISK_IO_INIT+FILE_IO+FILE_IO_INIT+VIRT_ALLOC+MEMINFO";

```

Comment:
Having Base+Latency is odd because they are both Kernel Groups and they overlap significantly. Base adds MEMINFO which is already explicitly included, and MEMINFO_WS which I have not found to be useful. If MEMINFO_WS is needed then make sure it works with the public version of WPA - I think the display of its data might be internal Microsoft only.